docker/config_template.yaml

```diff
@@ -13,6 +13,19 @@ persistence:
     datastores:
         {{- $db := default .Env.DB "cassandra" | lower -}}
         {{- if eq $db "cassandra" }}
+        default:
+            cassandra:
+                hosts: {{ default .Env.CASSANDRA_SEEDS "" }}
+                keyspace: {{ default .Env.KEYSPACE "cadence" }}
+                user: {{ default .Env.CASSANDRA_USER "" }}
+                password: {{ default .Env.CASSANDRA_PASSWORD "" }}
+        visibility:
+            cassandra:
+                hosts: {{ default .Env.CASSANDRA_SEEDS "" }}
+                keyspace: {{ default .Env.VISIBILITY_KEYSPACE "cadence_visibility" }}
+                user: {{ default .Env.CASSANDRA_USER "" }}
+                password: {{ default .Env.CASSANDRA_PASSWORD "" }}
+        {{- else if eq $db "scylla" }}
```

Comment:
Nit: This looks redundant. 
Can you help us  figure out how to improve this by using OR operator? I think it's using this templating language: https://github.com/jwilder/dockerize

Reply:
I simplified this.